mods/taskbar-clock-customization.wh.cpp

```diff
@@ -204,13 +208,25 @@ int StringCopyTruncated(PWSTR dest, size_t destSize, PCWSTR src, bool* truncated
     return i;
 }
 
+LPBYTE webContentsLast;
+int webContentsIndex = 0;
+
 void UpdateWebContent()
 {
     DWORD dwLength;
     LPBYTE pUrlContent = GetUrlContent(g_settings.webContentsUrl, &dwLength);
     if (!pUrlContent) {
         return;
     }
+    if (g_settings.webContentsIterate) {
+        if (webContentsLast && pUrlContent == webContentsLast) {
```

Comment:
`pUrlContent == webContentsLast` compares two pointers, not the content. In this case it will always be false. Also, the page might be different every time (e.g. if there's a timestamp or dynamic content) while the fetched items might not change, so perhaps it's better to extract the items and compare them.